search-sentiments-map/routes/trends.js

```diff
@@ -256,5 +256,10 @@ function getGlobalTrends(trendsByCountry) {
   return globalTrends;
 }
 
+// Necessary for unit testing.
+const trends = {
+  deleteAncientTrend,
+}
+
 module.exports.router = router;
-module.exports.updateTrendsFunction = updateDailyTrends;
\ No newline at end of file
+module.exports.trends = trends;
```

Comment:
If you are changing this here, be sure to also change the update-data.js file to call trends.updateTrendsFunction, and also add updateTrendsFunction into your trends const right above. Alternatively, don't delete the updateTrends export, and handle when you merge in.

Reply:
yea I deleted it bc i saw in chenyu's PR that she had updated these kind of things so I was going to merge after her PR is approved

Reply:
Okay soudns good!